test/index.js

```diff
@@ -40,4 +40,68 @@ describe('helmet', function () {
       pkg.restore()
     })
   })
+
+  it('chains all default middleware with empty options', function () {
+    moduleNames.forEach(function (moduleName) {
+      sinon.spy(helmet, moduleName)
+    })
+    helmet({})
+    moduleNames.forEach(function (moduleName) {
+      var pkg = helmet[moduleName]
+      var isDefault = config.defaultMiddleware.indexOf(moduleName) !== -1
+      if (isDefault) {
+        assert(pkg.calledOnce, moduleName + ' is default but is not called')
+      } else {
+        assert(!pkg.called, moduleName + ' is called but is not default')
+      }
+      pkg.restore()
+    })
+  })
+
+  it('chains nothing when disabling all default middleware', function () {
+    var options = {}
+    moduleNames.forEach(function (moduleName) {
+      sinon.spy(helmet, moduleName)
+    })
+    config.defaultMiddleware.forEach(function (moduleName) {
+      options[moduleName] = false
+    })
+    helmet(options)
+    moduleNames.forEach(function (moduleName) {
+      var pkg = helmet[moduleName]
+      var isDefault = config.defaultMiddleware.indexOf(moduleName) !== -1
+      if (isDefault) {
```

Comment:
This conditional can be simplified to `assert(!pkg.called, moduleName + ' should not be called')` because they both assert the same thing.